docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -39,6 +100,10 @@ The Fairlearn package is most applicable to two kinds of harms:
   information are extended or withheld. Examples include varying accuracy in
   face recognition, document search, or product recommendation.
 
+Other harms include stereotyping, denigration, and over- or
+underrepresentation. They are not mutually exclusive; a single system can
```

Comment:
"Note that harms are not mutually exclusive"

Reply:
Would it be helpful to remove this list and link to something like: [Harms Modeling](https://docs.microsoft.com/en-us/azure/architecture/guide/responsible-innovation/harms-modeling/) or [Assessing Harm: A Guide for Tech Builders](https://opdhsblobprod04.blob.core.windows.net/contents/f4438a49b5d04a4b93b0fa1f989369cf/8db74d210fb2fc34b7d6981ed0545adc?sv=2018-03-28&sr=b&si=ReadPolicy&sig=T3QsKIOzA7tO%2FQG1eo6UNxvs4369lSfYKrtr2MRCGqA%3D&st=2020-05-20T16%3A29%3A21Z&se=2020-05-21T16%3A39%3A21Z)?  My read is that the examples and guidelines here are mostly agnostic to the specific work within fairlearn.

Reply:
I think linking to other resources has value. Still, our documentation shouldn't just consist of links elsewhere, because most people are not going to spend the time to read multiple papers/articles/guides. The point of this guide is to provide a core set of things that shouldn't be misrepresented. We've told contributors on many occasions that they have to abide by certain guidelines around how to write/talk about fairness without ever specifying them. This could either be in the form of a list of rules (as it was in an earlier version of this PR) or something more like a text (as it is right now), but it can't be too much content (e.g. 10 papers/articles), otherwise people will simply give up.

[@riedgar-ms I'll address that one. I don't think the follow-up comment is related]

Reply:
Miro will rewrite this section of the user guide in a PR of his own, so I'll just put the list into the contributor guide. Reverting the last few commits and adjusting the lists according to earlier feedback.